xml/2018-10-14.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<email title="Weekly Digest">
+
+  <image
+    width="80"
+    src="https://i.imgur.com/5rJTZsq.jpg"
+    alt="Photo of digestive biscuits on a plate"
+    title=""/>
+  <hr/>
+
+  <text>
+    Hi again! It's time for another digest.
+
+    In the past week, we held four events: the Mums and Dads meeting, Linux talk,
+    Sponsors' Exhibition and the Bloomberg Decentralised App Development workshop.
+    These events were an overwhelming success with a total attendee count over all
+    four events of 1000. Thanks to everyone who participated and a special thanks
+    to all of our volunteers!
+
+    If you have any feedback about our events, feel free to [get in touch](mailto:docsoc@imperial.ac.uk).
+    Additionally, if you want to help out at our events, be sure to join our
+    [DoCSoc: Operations](https://www.facebook.com/groups/1509850436007222/) Facebook group.
+
+    In the coming week, we have our first inaugural bar night of the year, the
+    Welcome Bar Night, so stay tuned for more information and opportunities from
+    our sponsors.
+  </text>
+
+  <hr/>
+
+  <heading>Welcome Bar Night</heading>
+  <event
+    startDate="2018-10-18T19:00:00+01:00"
+    endDate="2018-10-18T23:00:00+01:00"
+    location="Metric">
+    <image src="https://i.imgur.com/TPhybmZ.jpg" />
+    <text>
+      It's time for DoCSoc's first bar night of the year: The Welcome Bar Night!
+
+      Come along for free drinks and to say hi to our new freshers or catch up
+      with friends - it's sure to be a night you don't want to miss out on.
+
+      Drinks tokens will be given upon entry for FREE to all DoCSoc members.
```

Comment:
At the event we will also announce the winners of our End of Year Survey, of the Speed Typing competition at the Freshers Fair and of the Mums and Dads Checklist!